sunkit_pyvista/plotter.py

```diff
@@ -509,6 +510,31 @@ def color_func(field_line):
 
         self._add_mesh_to_dict(block_name="field_lines", mesh=spline)
 
+    def plot_current_sheet(self, pfss_out, **kwargs):
+        """
+        Plot current sheet "(Br=0)".
+
+        Parameters
+        ----------
+        pfss_out : `sunkit_magex.pfss.output.Output`
+            Magnetic field calculated by `sunkit_magex`.
+        **kwargs :
+            Keyword arguments are passed to `pyvista.Plotter.add_mesh`.
+        """
+        sc_vect = pfss_out.grid.sc
```

Comment:
Is the way we are loading the output here the same as in #72?

Reply:
Not sure but it should be looked at. I would hope we can that code instead. 

If we can expand the code in 72 to work for all frames and return all the surfaces, we can then change this to plot a surface based on user input.